content/docs/1.62/operator.md

```diff
@@ -69,7 +69,7 @@ Make sure your `kubectl` command is properly configured to talk to a valid Kuber
 To install the operator, run:
 ```bash
 kubectl create namespace observability # <1>
-kubectl create -f https://github.com/jaegertracing/jaeger-operator/releases/download/v{{< currentVersion >}}.0/jaeger-operator.yaml -n observability # <2>
```

Comment:
I there really only one reference to version in the doc? I thought I saw several.